packages/multichain/src/scope/constants.ts

```diff
@@ -0,0 +1,44 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+
+import type { NonWalletKnownCaipNamespace } from './types';
+
+// ScopeStrings for offchain methods that are not specific to a chainId but are specific to a CAIP namespace
```

Comment:
Nit: This comment won't be picked up by IntelliSense or other documentation tooling because the `//` style comment isn't considered to be associated with the next line.

Convert these to TSDoc-style block comments, and the description will follow the enums around

Reply:
Added a bunch more TSDocs here: https://github.com/MetaMask/core/pull/4784/commits/6e345cf0c4a13329261720bf02ba6138538fa309